components/pricing/PricingCard.vue

```diff
@@ -41,12 +42,21 @@ export default {
     amount: Number,
     features: Object,
     isHighlighted: Boolean,
+    isYearly: Boolean,
   },
   data() {
     return {
       highlight: this.isHighlighted,
     }
   },
+  computed: {
+    yearlyPrice(){
+      return this.amount * 11;
+    },
+    originalPrice(){
+      return this.amount * 12;
+    }
```

Comment:
This should not work like this. As in future we may change this logic. This computation was something that we used now but maybe it will not be the same in future.

So better to create an array of plan and send that inside and not rely on computaion on the fronend side.

you can create keys in an object { price: '', orignalPrice: '' }